run_dir/static/js/data_del_plot.js

```diff
@@ -0,0 +1,357 @@
+                    const dm = 2 < 0 ? 0 : 2;
+                    const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+                    const i = Math.floor(Math.log(sum) / Math.log(k));
+                    var c_sum = parseFloat((sum / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
+                this.setTitle({text: "File size of projects, sum in bytes: " + c_sum}, false, false)
+                }
+            }
+        },
+        title: {
+            text : name+' of projects'
+        },
+        yAxis: {
+            min : 0,
+            title : {
+              text : name
+            }
+        },
+        legend: {
+            title: {
+                text: 'Click to hide:',
+                align: 'center'
+                }
+        },
+        plotOptions : {
+            series : {
+                turboThreshold: 0,
+                point: {
+                  events: {
+                    click: function() {
+                      window.open(this.options.ownURL)
+                    }
+                  }
+                }
+            }
+        },
+        tooltip: {
+            shared: true,
+            useHTML: true,
+            headerFormat: '<span style="color:{point.color}">\u25CF</span><small>{point.key}</small><br />',
+            pointFormatter: function () {
+                if (this.y === 0) return '0 Bytes';
+                const k = 1024;
+                const dm = 2 < 0 ? 0 : 2;
+                const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+                const i = Math.floor(Math.log(this.y) / Math.log(k));
+                var fs = parseFloat((this.y / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
+                return this.series.name + ', ' + this.pm + ': ' + '<b>' + fs + '</b>'
+            },
+        },
+        credits: {
+            enabled: false
+        },
+        xAxis: {
+            type: 'category',
+            labels: {
+                enabled: false,
+            },
+            title : {
+                text : 'Close Date'
+            },
+            categories: []
+        },
+        series: [{
+            name: name,
+            data: []
+        }],
+        exporting: {
+          csv: {
+            itemDelimiter: ';'
+          }
+        }
+    };
+    serie = build_series(window.current_plot_data, key, name, view_type, filter_inst_type);
+    toplot.series = serie[1];
+    toplot.xAxis.categories = serie[0];
+    $("#main_plot").highcharts(toplot);
+    window.current_plot_obj = toplot;
+}
+
+function build_series(data, key, name, view_type, filter_inst_type){
+    var series = [];
+    var view_color = "";
+    var series_name = "";
+    var categories = [];
+    for (d in data){
+        var bioinfo_link="/bioinfo/"+data[d][0];
+        var project_name = data[d][1].project_name;
+        var date_close = data[d][1].close_date;
+        if (data[d][1].sequencing_platform == null){
+            continue;
```

Comment:
I'm not sure that we want to ignore any data where the sequencing platform is not set? Better to have an "everything else" category instead I think.